catalog/app/containers/Admin/Buckets.tsx

```diff
@@ -957,7 +1172,14 @@ interface CRUDProps {
 }
 
 function CRUD({ bucketName }: CRUDProps) {
-  const [{ data }] = urql.useQuery({ query: BUCKET_CONFIGS_QUERY })
+  const [{ error, data }] = urql.useQuery({ query: BUCKET_CONFIGS_QUERY })
+  if (!data && error) throw error
+
+  const sentry = Sentry.use()
+  React.useEffect(() => {
```

Comment:
This, I think, violates Rules of Hooks too

Reply:
it does, but when it happens it means an application error, not something we should recover from